stable/launcher/powershell-wrapper

```diff
@@ -5,4 +5,7 @@
 # Create $XDG_RUNTIME_DIR if it doesn't exist
 [ -n "$XDG_RUNTIME_DIR" ] && mkdir -p $XDG_RUNTIME_DIR -m 700
 
+# make sure DOTNET loads the ICU that we ship
+export CLR_ICU_VERSION_OVERRIDE=60.2
```

Comment:
Does this need to be exported?

It might be better to set the env var with the exec, [like this](https://stackoverflow.com/a/22869763)?

Reply:
This script is already being exec'ed.  It seems redundant.

Reply:
But found there is a YAML syntax to put it in that exec...  Moving it there.